metagamer/environments/tictactoe.py

```diff
@@ -0,0 +1,176 @@
+"""
+This file provides some simple tools to simulate and work with a tic tac toe board.
+The functions provided assume the board is represented by
+    - np array of shape 3,3
+    - player 1 is 1, player 2 is -1 and empty is 0
+
+
+"""
+import logging
+
+import numpy as np
+import gym
+from gym import spaces, error
+
+from typing import Iterable, Any, Tuple, Dict, List, Optional, Iterator
```

Comment:
This seems curious, all these classes have a special meaning in 'typing'?  I'll see if I can figure out as I read on.

Reply:
You'll probably need to read up on typing in python. But long and short they do have special meanings.